lib/pages/pressable/pressable-sites-page.js

```diff
@@ -1,8 +1,50 @@
-import { By as by } from 'selenium-webdriver';
+import { By } from 'selenium-webdriver';
+import config from 'config';
+
 import BaseContainer from '../../base-container';
+import PressableSiteSettingsPage from './pressable-site-settings-page';
+
+import * as driverHelper from '../../driver-helper';
+
+const explicitWaitMS = config.get( 'explicitWaitMS' );
 
 export default class PressableSitesPage extends BaseContainer {
-	constructor( driver ) {
-		super( driver, by.css( 'form[action="/sites"]' ) );
+	constructor( driver, visit = false ) {
+		const url = 'https://my.pressable.com/sites';
+
+		super( driver, By.css( 'form[action="/sites"]' ), visit, url );
```

Comment:
This form only appears when the account hasn't reached its maximum number of sites. If the account has reached its maximum sites, and you try to use this page (e.g. to delete a site), it fails on the `waitForPage()` method:

`TimeoutError: Timed out waiting for element with css selector of 'form[action="/sites"]' to be present and displayed`

It looks like the class `site-show-sections` may be specific to this page and could be used here instead.